cratedb_fivetran_destination/main.py

```diff
@@ -11,7 +16,24 @@
 SEVERE = "SEVERE"
 
 
+class Knowledge:
+    field_map = {
+        "_fivetran_id": "__fivetran_id",
+        "_fivetran_synced": "__fivetran_synced",
+        "_fivetran_deleted": "__fivetran_deleted",
+    }
+
+    # FIXME: Add more type mappings.
+    type_map = {
+        DataType.INT: sa.Integer(),
+    }
+
+
 class CrateDBDestinationImpl(destination_sdk_pb2_grpc.DestinationConnectorServicer):
+    def __init__(self):
+        self.metadata = sa.MetaData()
+        self.engine = sa.create_engine("crate://")
```

Comment:
Needs to use actual connection parameters.

Reply:
Done. The code now uses the `url` form field for configuring the SQLAlchemy database URL, nothing else by now.

https://github.com/crate-workbench/cratedb-fivetran-destination/blob/925bae299ea34fd33316eb64a9634cdb8a19fa0b/cratedb_fivetran_destination/main.py#L52-L59